examples/kubernetes-egress-gateway/egress-ip-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "egress-ip-assign"
+  labels:
+    name: "egress-ip-assign"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: "egress-ip-assign"
+  template:
+    metadata:
+      labels:
+        name: "egress-ip-assign"
+    spec:
+      hostNetwork: true
+      containers:
+      - name: egress-ip
+        image: docker.io/library/busybox:1.31.1
+        command: ["/bin/sh","-c"]
+        securityContext:
+          privileged: true
+        env:
+        - name: EGRESS_IPS
+          value: "192.168.33.100/24 192.168.33.101/24"
+        args:
+        - "for i in $EGRESS_IPS; do ip address add $i dev enp0s8; done; sleep 10000000"
```

Comment:
Is this going to be enough for users running in e.g. GKE, EKS, AKS?

Reply:
This is the guide for onprem environment. In cloud, the way to assign IP addresses will be very different in different cloud providers. We haven't covered those environments yet. Maybe we could do it as next step.

Reply:
Maybe it would be enough to use the existing node IP for managed Kubernetes? It won't demonstrate the SNAT part of the egress gateway, but at least users will be able to observe the redirect. I guess it's better than nothing.

If that's not an option, we should at least clarify at the top that the guide doesn't currently support managed Kubernetes. I would be very careful how we word this though; we don't want to give the impression the feature doesn't support managed Kubernetes.

Reply:
Make sense. I'll try in on GKE first as an example. But I won't be able to cover other platforms.